com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,202 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.graphics.Color
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RelativeLayout
+import com.chartboost.heliumsdk.ad.HeliumAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.ad.HeliumFullscreenAd
+import com.chartboost.heliumsdk.ad.HeliumRewardedAd
+import com.unity3d.player.UnityPlayer
+
+class HeliumUnityAdWrapper(private val ad: HeliumAd) {
+    /**
+     * This is the container for the banner.
+     * We need a relative layout to position the banner in one of the 7 possible positions.
+     * HeliumBannerAd is just a ViewGroup.
+     * */
+    private var bannerLayout: RelativeLayout? = null
+    private val activity: Activity? = UnityPlayer.currentActivity
+
+    fun load() {
+        HeliumUnityBridge.runTaskOnUiThread {
+            ad.load()
+        }
+    }
+
+    fun load(screenLocation: Int) {
+        HeliumUnityBridge.runTaskOnUiThread {
+            createBannerLayout(screenLocation)
+            load()
+        }
+    }
+
+    fun show() {
+        HeliumUnityBridge.runTaskOnUiThread {
+            if (ad is HeliumFullscreenAd) {
+                ad.show()
+            }
```

Comment:
Do we want to log something otherwise?

Reply:
Same as the other comment above, there isn't really any scenario where this is called by anything else than a full screen ad. No need to log.